### PR TITLE
Add code snippet to add a country

### PR DESCRIPTION
**Pull request by nielslange**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woodocs/issues/102

This PR adds the code snippet [Add a country](http://www.woocommerce.com/document/add-a-new-country-to-countries-list/) to the Woo docs.

### How to test the changes in this Pull Request:

1. Compare the content of the file docs/snippets/add-a-country.md with the content on http://www.woocommerce.com/document/add-a-new-country-to-countries-list/.
2. Verify that the new link within docs/snippets/README.md points to the newly added code snippet.

### CODE REVIEW

File: docs/snippets/add-a-country.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Add a country
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
```
Comment (barryhughes) [inline docs/snippets/add-a-country.md]: Similar to related PRs, we can drop this call-out.

```suggestion
```

**Review by barryhughes — CHANGES_REQUESTED** [review]
Looks good, confirmed the snippet works. Left a note about a minor adjustment we should probably do.

File: docs/snippets/add-a-country.md (hunk 1 above)
Comment (nielslange) [reply docs/snippets/add-a-country.md]: Removing this note based on p1692723749470709/1692723559.960259-slack-C0516GKR69W

**Comment by nielslange** [comment]
> Looks good, confirmed the snippet works. Left a note about a minor adjustment we should probably do.

Thanks for reviewing this PR, @barryhughes. Unfortunately, I do not have the necessary permissions to merge this PR without your approval. I applied all your suggestions. Could you double-check and approve this PR so that I can merge it?

**Review by barryhughes — APPROVED** [review]

**Pull request merged**